task07/server.ts

```diff
@@ -0,0 +1,32 @@
+import express from 'express';
+import mongoose from 'mongoose';
+import { handleErrorMiddleware } from './error/middlewares';
+import { AppError } from './error/appError';
+import { HttpStatusCode } from './error/statusCode';
+import globalRoute from '../task06/routes/global.route';
+
+const PORT = 8000;
+const uri: string =
+  'mongodb://root:nodegmp@127.0.0.1:27017/mydatabase?authSource=admin';
```

Comment:
FYI: It's better to move this part to some sort of a config and receive such values from the environment vars.